xml/schema/Contact/Group.xml

```diff
@@ -176,6 +176,15 @@
     <fieldName>cache_date</fieldName>
     <add>5.34</add>
   </index>
+  <field>
+    <name>cache_fill_took</name>
+    <type>decimal</type>
```

Comment:
I think you want float here, because decimal gets converted to CRM_Utils_Type::T_MONEY.

Reply:
and double for the upgrade